### ISSUE TITLE
Generate Cross-val folds

### ISSUE THREAD

**Issue by dvetal** [ayota/ddl_nlp#3]
We need to develop a file that parses the downloaded text and returns a training and a test set k-1 times.  The next step in the process (gensim word2vec training) should be able to pull just the training data to conduct training.  The function should also return test data that is accessible to our evaluation tasks.

**Comment by dvetal** [comment]
PR #5 resolves this issue.
